components/Pagination/sub-components/Desktop.jsx

```diff
@@ -89,6 +91,7 @@ Desktop.defaultProps = {
   prevButtonText: 'Previous',
   handlePageClick: undefined,
   followOnlyFirstPage: false,
+  showLastPagination: true,
```

Comment:
was this value already declared in Pagination.jsx?
i think its redundant here.

Reply:
I do